covid_graphs/covid_graphs/predictions.py

```diff
@@ -9,7 +9,9 @@
 @dataclass(frozen=True, eq=True)
 class PredictionEvent:
     name: str
+    # Data until this date (inclusive) was used for the prediction.
     date: datetime.date
+    creation_date: datetime.date
 
```

Comment:
The comment saves the day for not very usefull name. I'd consider something like `date` -> `last_data_date` and `creation_date` -> `prediction_date` or simply `date` as it's already part of PredictionEvent class. Until now, I was thinking that the old `date` was date of prediction and not last data date used for prediction :smile:

Reply:
Agree here

Reply:
`last_data_date` is a great naming suggestion. I'll make the change (also creation_date -> prediction_date).